server/tasks/create_device_inventory.json

```diff
@@ -2,11 +2,12 @@
     {
         "name": "create_device_inventory",
         "retryCount": 5,
-        "timeoutSeconds": 10,
+        "timeoutSeconds": 120,
```

Comment:
2 minutes sounds like a long time for creating device inventory, don't you think?

Reply:
it's not a timeout for creating device inventory
according to https://netflix.github.io/conductor/configuration/taskdef/#timeout-policy it's

> Time in milliseconds, after which the task is marked as TIMED_OUT if not completed after transitioning to IN_PROGRESS status for the first time

actually, it's not in milliseconds, but seconds (according to https://tracker.mender.io/browse/MC-637)
also, I have used values from MC-637, only rounded to minutes

still, if you have suggestions on how this configuration should look like, feel free to add them in the comments

Reply:
yes, it's timeout to start re-trying

In mentioned ticket re-tries are related to sending emails and I believe it might be even 5 or 10 minutes because we interact with external system (mail server) and task can fail because of it, so, some waiting before next try is good idea, because external service can be restore (and this is probably expected) during that time.
But when we talk about create device inventory this is internal interaction and timeouts might be different because if something doesn't work internally we need to react and fix it asap. The reaction is based on failed states and with big timeouts it comes to failed state with a delay and we react with a delay as well.

I believe that timeout in 10 seconds is very ok for internal interactions the only thing is needed here is "responseTimeoutSeconds" should be in between 0 and 10 (maybe 5).

Reply:
right; thanks for the comment
I've updated timeouts here and in the delete_device_deployments task

Reply:
I guess this is applicable to all tasks for communication with core services (delete_device_inventory in this PR and create_tenant, delete_tokens_deviceauth, delete_tokens_useradm in https://github.com/mendersoftware/mender-conductor-enterprise/pull/30)

Reply:
done